deploy/ansible/deploy-legion-enclave.yml

```diff
@@ -12,9 +12,9 @@
 
 - hosts: localhost
   connection: local
-  gather_facts: False
+  gather_facts: True
```

Comment:
why do we need facts gathering enabled?

Reply:
There were some ustable behaivor. This flag doesn't affect anything else, so I suppoose to keep this as True, at least for now